src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -82,11 +105,11 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Site site = (Site) o;
-        return id == site.id && name.equals(site.name) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && created == site.created;
+        return id == site.id && name.equals(site.name) && description.equals(site.description) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && visible.equals(site.visible) && created == site.created;
```

Comment:
Since `visible` and `description` will be null for existing objects, it may potentially raise a NPE here. Maybe better to check for null value for visible and description?

Reply:
Possibly no need for `visible` null check if we default it